content/trial_getting_started/set_trial_manager/download_community.md

```diff
@@ -36,5 +36,7 @@ ____
 
 What's next?
 
-* Optional - When using a remote Docker service, working with Cloudify using a local CLI is the recommended approach. [Install local Cloudify CLI]({{< relref "/install_maintain/installation/installing-cli.md" >}}).
-* Go ahead and examine our [example based tutorials]({{< relref "trial_getting_started/examples/_index.md" >}}) to learn about Cloudify basics.
+* To test your installation using the command line utility on your docker image refer to the [local CLI guide] ({{< relref "trial_getting_started/set_trial_manager/getting-started-with-cloudify-docker-and-cli" >}})
```

Comment:
I would consider replacing "To test" with "To operate" or "To manage".

Reply:
done